composer.json

```diff
@@ -7,16 +7,22 @@
         }
     },
     "require": {
-        "php": ">=5.5",
-        "symfony/symfony": "^2.6|3.*",
-        "paysera/lib-serializer": "^2.0.2",
-        "doctrine/orm": "^2.0"
-    },
-    "require-dev": {
-        "phpunit/phpunit": "^4.8",
-        "mockery/mockery": "^0.9.4"
+        "php": "^7.0",
+        "paysera/lib-serializer": "^3.0",
```

Comment:
Is 2.* support really dropped? It's better to leave `^2.6|^3.0` if it's still supported

Reply:
Why bother with Symfony 2? It's not supported any more - I've removed support for it.